src/sandstorm/app-index/indexer.c++

```diff
@@ -389,7 +394,7 @@ void Indexer::updateIndexInternal(kj::StringPtr outputFilename, kj::StringPtr ou
     kj::FdOutputStream(file.getFd()).write(text.begin(), text.size());
     file.finalize(kj::str(outputDir, "/", appEntry.first, ".json"));
 
-    if (approvedApps) {
+    if (!experimental) {
```

Comment:
In particular, it's a bit weird to use `/var/apps` within this. Is there an analogous thing for experimental apps?